doc/user/content/demos/microservice.md

```diff
@@ -436,7 +436,7 @@ can see that Materialize is ingesting the `protobuf` data and normalizing it.
 1. Launch `psql` (or any Materialize CLI client), connecting to `materialize`:
 
     ```shell
-    psql -U materialize -h localhost -p 6875 -d materialize
+    psql -U materialize -h localhost -p $(./mzcompose list-ports materialized) -d materialize
```

Comment:
do we still need to specify `-d materialize`? (not your change i know)

Reply:
We do! I added a `./mzcompose sql` command to make this *much* easier though.